idea/resources-en/messages/KotlinBundle.properties

```diff
@@ -1628,6 +1628,9 @@ choose.use.site.target=Choose use-site target
 add.use.site.target=Add use-site target
 add.braces.to.0.statement=Add braces to ''{0}'' statement
 add.braces.to.when.entry=Add braces to 'when' entry
+add.braces.to.all.branches=Add braces to all branches
+add.braces.to.if.all.statements=Add braces to 'if' all statements
+add.braces.to.when.all.entries=Add braces to 'when' all entries
```

Comment:
Should these be "all 'if' statements"/ "all 'when' entries" instead of the current version?